src/main/java/com/kllhy/roadmap/travel/presentation/TravelController.java

```diff
@@ -1,8 +1,50 @@
 package com.kllhy.roadmap.travel.presentation;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.kllhy.roadmap.common.response.ApiResponse;
+import com.kllhy.roadmap.common.response.SuccessCode;
+import com.kllhy.roadmap.travel.application.service.command.TravelService;
+import com.kllhy.roadmap.travel.application.service.query.TravelQueryService;
+import com.kllhy.roadmap.travel.application.view.TravelView;
+import com.kllhy.roadmap.travel.presentation.request.TravelCreateRequest;
+import com.kllhy.roadmap.travel.presentation.request.TravelUpdateRequest;
+import com.kllhy.roadmap.travel.presentation.response.TravelResponse;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/api-v1/travel")
```

Comment:
저희 api 스펙 `api-v1/travels` 복수형으로 통일하는게 좋을 것 같습니다!
```suggestion
@RequestMapping("/api-v1/travels")
```